openshift/templates/identity-kit-public/identity-kit-public-deploy.bc.param

```diff
@@ -4,17 +4,23 @@
 # Template File: ../openshift/templates/identity-kit-public/identity-kit-public-deploy.json
 #=========================================================
 NAME=identity-kit-public
+APP_NAME=identity-kit
 APP_GROUP=identity-kit
+SUFFIX=-bc
 IMAGE_NAMESPACE=devex-von-tools
 CADDY_CONFIG_FILE_NAME=Caddyfile
 CADDY_CONFIG_MAP_NAME=identity-kit-public-caddy-conf
 CADDY_CONFIG_MOUNT_PATH=/etc/
+APPCONFIG_CONFIG_FILE_NAME=config.json
+APPCONFIG_CONFIG_MAP_NAME=identity-kit-admin-appconfig
+APPCONFIG_CONFIG_MOUNT_PATH=/srv/assets/config/
 WEB_HOST_NAME=0.0.0.0
 WEB_HOST_PORT=8080
 WEB_HOST_TLS=off
 API_HOST=identity-kit-controller
 API_PORT=8080
-APPLICATION_DOMAIN=identity-kit.pathfinder.gov.bc.ca
```

Comment:
I don't think you meant to make this change.

Reply:
This is intended: the `prod` deployment file hasthe right host, so I kept the base configuration generic.